Add Netgear allow/block switch (#21528)
home-assistant/home-assistant.github.io: source/_integrations/netgear.markdown

ha_ssdp: true
---

This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.

> Both routers and acces points can be used with this integration, some acces points will not be automatically discoverd and need to be set up manually.

Reviewer: ```suggestion
Both routers and access points can be used with this integration. Some access points will not be automatically discovered and need to be set up manually.
```

{% include integrations/config_flow.md %}

## Optional settings

> The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration.

Reviewer: ```suggestion
The optional configuration settings of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration.
```

> The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration and can be accesed as followes: in the sidebar of your Home Assistant instance click on "Configuration" -> "Integrations" -> find the NETGEAR integration and click "Configure".

Reviewer: Maybe just remove this whole section? The location of an options flow is the same for all integrations that support that.
```suggestion
The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration and can be accessed as follows: in the sidebar of your Home Assistant instance click on "Configuration" -> "Integrations" -> find the NETGEAR integration and click "Configure".
```

Author: Sure I think you are right, I will remove it.

Author: done

> ### Consider_home

Reviewer 2: ```suggestion
### Consider_home

```
After each header, first a white line and then text

Author: done

The 'consider_home' time is the amount of seconds to wait till marking someone as not home after not being seen. This parameter is most useful for households with Apple iOS devices that go into sleep mode while still at home to conserve battery life. iPhones will occasionally drop off the network and then re-appear. consider_home helps prevent false alarms in presence detection.

## Router entities
The NETGEAR router will have the following entities:

### Traffic meter data
The total and average amount of downloaded/uploaded data through the router can be tracked per day/week/month.

> In order for these entities to display the data (instead of 0), the "Traffic Meter" schould be enabeld in the router settings.

Reviewer: ```suggestion
In order for these entities to display the data (instead of 0), the "Traffic Meter" schould be enabled in the router settings.
```

Log into your router > Select ADVANCED > Advanced Setup > Traffic Meter > Enable Traffic Meter check box.

## Connected device entities
For each device connected to the NETGEAR router the following entities will be available:

### Device tracker
Displays if the device is currently connected to the router (Home) or not (Away).

### Allowed on Network
Switch that lets you Allow or Block a device on the Network.
For this entity to actually Block the device, "Access Control" needs to be turned on in the Router settings.

> Log into your router > Select ADVANCED > Security > Access Control > Turn on Access Control check box.

Reviewer 2: ```suggestion
Log into your router > Select **ADVANCED** > **Security** > **Access Control** > Turn on Access Control check box.
```

Author: thanks

### Signal strength
Displays the wifi signal strength of the device

### Link rate

> Displays the current link rate of the device indicating the maximum posible data speed with the current connection.

Reviewer: ```suggestion
Displays the current link rate of the device indicating the maximum possible data speed with the current connection.
```

ha_ssdp: true
---

This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.
Both routers and access points can be used with this integration. Some access points will not be automatically discovered and need to be set up manually.

{% include integrations/config_flow.md %}

> ## Optional settings

Reviewer 3: We have `{% include integrations/option_flow.md %}` for that nowadays, please check e.g., the `cast` integration documentation page on how to use it.

Author: brilliant, that is perfect, thank you!

ha_ssdp: true
---

This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.
Both routers and access points can be used with this integration. Some access points will not be automatically discovered and need to be set up manually.

{% include integrations/config_flow.md %}

{% include integrations/option_flow.md %}
{% configuration_basic %}
Consider_home:

>   description: "The 'consider_home' time is the amount of seconds to wait till marking someone as not home after not being seen. This parameter is most useful for households with Apple iOS devices that go into sleep mode while still at home to conserve battery life. iPhones will occasionally drop off the network and then re-appear. consider_home helps prevent false alarms in presence detection."

Reviewer 3: ```suggestion
  description: "The consider home time is the number of seconds to wait till marking someone as not home after not being seen. This parameter is most useful for households with Apple iOS devices that go into sleep mode while still at home to conserve battery life. iPhones will occasionally drop off the network and then re-appear. This option helps prevent false alarms in presence detection."
```

{% endconfiguration_basic %}

## Router entities
The NETGEAR router will have the following entities:

### Reboot button

Button entity to restart the router.

### Traffic meter data

The total and average amount of downloaded/uploaded data through the router can be tracked per day/week/month.

> In order for these entities to display the data (instead of 0), the "Traffic Meter" schould be enabled in the router settings.

Reviewer 3: ```suggestion
In order for these entities to display the data (instead of 0), the "Traffic Meter" should be enabled in the router settings.
```

Log into your router > Select **ADVANCED** > **Advanced Setup** > **Traffic Meter** > **Enable Traffic Meter** check box.

> ## Connected device entities
> For each device connected to the NETGEAR router the following entities will be available:

Reviewer 3: ```suggestion
## Connected device entities

For each device connected to the NETGEAR router the following entities will be available:
```

### Device tracker

Displays if the device is currently connected to the router (Home) or not (Away).

### Allowed on Network

Switch that lets you Allow or Block a device on the Network.
For this entity to actually Block the device, "Access Control" needs to be turned on in the Router settings.
Log into your router > Select **ADVANCED** > **Security** > **Access Control** > **Turn on Access Control** check box.

### Signal strength

> Displays the wifi signal strength of the device

Reviewer 3: ```suggestion
Displays the wifi signal strength of the device.
```
